core/PRP/Region/hsBounds.cpp

```diff
@@ -176,6 +176,16 @@ hsBounds3Corners hsBounds3::getCorners() const
     return corners;
 }
 
+void hsBounds3::setFromPoints(size_t numPoints, const hsVector3* points)
+{
+    if (numPoints > 0) {
+        setMins(points[0]);
+        setMaxs(points[0]);
+    }
```

Comment:
It seems to me that calling this method with `numPoints <= 0` is not useful, so I think the if statement could be removed.

Reply:
Wasn't sure whether this might be acceptable when dealing with exotic Blender scene setup. But yeah, probably better to let the caller handle zero-vertices cases.